<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageProfileEdit</class>
 <widget class="QWidget" name="PageProfileEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="selName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="selAge">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>44</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>110</number>
   </property>
   <property name="value">
    <number>18</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="selHeight">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>44</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height (cm)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="selWeight">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>44</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight (kg)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFinish">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
